server/routes/routes.js

```diff
@@ -1,21 +1,59 @@
 const express = require('express');
+const db = require('../../database/review.js');
 
 const router = express.Router();
 
+
 router.get('/restaurants/:restaurantId/reviews', (req, res) => {
-  res.json(['hello']);
+  db.findReviewsByRestaurant(req.params.restaurantId, (err, data) => (
+    err ? res.sendStatus(500) : res.json(data)
+  ));
 });
 
+
 router.post('/restaurants/:restaurantId/reviews', (req, res) => {
-  res.sendStatus(201);
+  if (!req.body.rating || !req.body.review) {
+    res.sendStatus(404);
+  } else {
+    const review = {
+      restaurant: req.params.restaurantId,
+      userName: req.body.user || 'anonymous',
+      userPhoto: req.body.photo || '',
+      userLocation: req.body.location || 'San Francisco, CA',
+      rating: req.body.rating,
+      date: new Date(),
+      reviewBody: req.body.review,
+      useful: 0,
```

Comment:
I'd consider a possible refactor of having these as default values in your schema and only sending them to your db insert function if they are being updated

Reply:
I like this suggestion, but currently I haven't decided how I want to handle the "default" user on the front end, (mostly if I'm going to allow anonymous users), so I want to hold off on refactoring the schema for now.